test/system/src/main/java/io/pravega/test/system/framework/services/PravegaSegmentStoreService.java

```diff
@@ -132,6 +136,7 @@ private App createPravegaSegmentStoreApp() {
         map.put("HDFS_URL", "hdfs.marathon.containerip.dcos.thisdcos.directory:8020");
```

Comment:
We should remove env vars already being set for HDFS, since it will be set using `SEGMENTSTORE_EXTRA_ENV` . Method `getCutomEnvVars()` could default to HDFS.

Reply:
Changed the PR to cover this. Thanks.